README.md

```diff
@@ -221,41 +229,45 @@ neogit.setup {
 ```
 
 ### List of status commands:
-
-- Close
-- InitRepo
-- Depth1 (Set foldlevel to 1)
-- Depth2 (Set foldlevel to 2)
-- Depth3 (Set foldlevel to 3)
-- Depth4 (Set foldlevel to 4)
-- Toggle
-- Discard (Normal and visual mode)
-- Stage (Normal and visual mode)
-- StageUnstaged
-- StageAll
-- GoToFile
-- Unstage (Normal and visual mode)
-- UnstageStaged
-- CommandHistory
-- RefreshBuffer
-- HelpPopup
-- PullPopup
-- PushPopup
-- FetchPopup
-- CommitPopup
-- LogPopup
-- StashPopup
-- BranchPopup
+* Close

```

Comment:
TODO: figure out a way to have this outputted from the code rather than manually keep it up to date